generated/nidigital/nidigital/session.py

```diff
@@ -896,7 +1299,7 @@ def configure_time_set_drive_edges2x(self, time_set_name, format, drive_on_edge,
     def configure_time_set_drive_format(self, time_set_name, drive_format):
         r'''configure_time_set_drive_format
 
-        TBD
+        Configures the drive format for the pins not specified in the **pinList**. Use this method to modify time set values after applying a timing sheet with the apply_levels_and_timing method, or to create time sets programmatically without the use of timing sheets. This method does not modify the timing sheet file or the timing sheet contents that will be used in future calls to apply_levels_and_timing; it only affects the values of the current timing context.
```

Comment:
"Configures the drive format for the pins not specified in the **pinList**."

"not" word shouldn't be there. (Curious to know how that snuck in there)

Reply:
The 'not' is in the C API Reference documentation.

Reply:
Done